onelogin/get.go

```diff
@@ -19,15 +19,20 @@ import (
 	"github.com/allcloud-io/clisso/log"
 	"github.com/allcloud-io/clisso/saml"
 	"github.com/allcloud-io/clisso/spinner"
+	"github.com/allcloud-io/clisso/yubikey"
 	"github.com/icza/gog"
 	"github.com/sirupsen/logrus"
+	"github.com/spf13/viper"
```

Comment:
When fixing conflicts, please note that `logrus` was encapsulated in our own log package.